pkg/phases.go

```diff
@@ -16,6 +16,7 @@ func init() {
 			phases.Environment,
 			phases.Containers,
 			phases.AnsiblePhase,
+                        phases.Kernel,
```

Comment:
Not sure if there is a tab/spacing issue here? Either way, run `gofmt` to fix any formatting issues, and `make lint` for any other linting issues

Reply:
just FYI, make lint has no target in this repo

Reply:
my bad, we should add one:

```Makefile
.PHONY: lint
lint: build
        which golangci-lint  2>&1 > /dev/null || curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
	golangci-lint run --verbose --print-resources-usage
```

Reply:
Addressed in https://github.com/flanksource/konfigadm/pull/100